tests/cassettes/test_export/test_export_api.yaml

```diff
@@ -0,0 +1,2940 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      User-Agent:
+      - python-requests/2.32.3
+    method: GET
+    uri: https://hirondelle.crim.ca/stac
+  response:
+    body:
+      string: '{"type": "Catalog", "id": "stac-fastapi", "title": "Data Analytics
```

Comment:
Not related to this PR, but I guess we will have to address that in birdhouse-deploy to avoid conflicting IDs between our instances...

Reply:
Yeah good point.